### DISCUSSION TITLE
Mimir Backup ??

### DISCUSSION THREAD

**Discussion by ShibraAmin18** [grafana/mimir#4889] [Help and support] [answered]
We have setup mimir in EKS using storage as S3 bucket. 

We need to restore the metrics in different or same cluster.

Can we use the same S3 bucket as it has the data? And will the old data metrics with old timestamp will be visible in grafana ?

**Comment by dimitarvdimitrov** [comment, marked as answer]
Yes, pointing a mimir cluster at the same bucket will discover all the blocks which are there and make them queriable. Normally it can take up to an hour for Mimir to discover the blocks. 

It's important that you don't have two sets of compactors compacting the same bucket. So if you have two clusters using the same bucket, try to make sure that they aren't both running at the same time.
